package.json

```diff
@@ -59,7 +59,8 @@
     "author": "reinierguerra",
     "license": "",
     "engines": {
-        "node": ">=10.15.1"
+        "node": "18.x",
+        "yarn": "1.22.19"
```

Comment:
This is a *big* update. Will definitely need to test in staging before deploying this to Salesforce's prod app.